proxy/InkAPI.cc

```diff
@@ -9212,6 +9213,19 @@ TSUuidStringGet(const TSUuid uuid)
   return nullptr;
 }
 
+const char *
+TSClientRequestUuidGet(TSHttpTxn txnp)
+{
+  TSUuid process = TSProcessUuidGet();
+  if (process) {
+    std::ostringstream oss;
+    oss << TSUuidStringGet(process) << '-' << TSHttpTxnIdGet(txnp);
+    return oss.str().c_str();
```

Comment:
You are retuning a pointer of the data inside of a locally scoped variable.  I would suggest doing a ats_malloc() since you know the size of the string and a memcpy() of the values.  Callers will need to call TSfree to free the memory.